lib/dirac_twisted_clover.cpp

```diff
@@ -169,6 +191,25 @@ namespace quda {
     twistedCloverApply(out, in, QUDA_TWIST_GAMMA5_INVERSE, parity);
   }
 
+  void DiracTwistedCloverPC::WilsonDslash(ColorSpinorField &out, const ColorSpinorField &in, QudaParity parity) const
+  {
+    if (in.TwistFlavor() == QUDA_TWIST_SINGLET) {
+      DiracWilson::DslashXpay(out, in, parity, in, 0.0);
+    } else {
+      ApplyDomainWall4D(out, in, *gauge, 0.0, 0.0, nullptr, nullptr, in, parity, dagger, commDim, profile);
```

Comment:
I'd appreciate a brief comment on why this is being used, just something simple like "Reusing domain wall machinery to apply WilsonDslash to both chiralities" or w/e is most clear

Reply:
d83e46f5ca2fdc4fec6cba4e396d3ef11e8e6917